versatileimagefield/templates/django/forms/widgets/attrs.html

```diff
@@ -0,0 +1 @@
+{% for name, value in widget.attrs.items %}{% if value != False %} {{ name }}{% if value != True %}="{{ value }}"{% endif %}{% endif %}{% endfor %}
```

Comment:
Yeah I was tempted to even just inline it in the template, but I'll just move it to the versatileimagefield template folder instead! Thanks for the review and sorry about a lot of the test / style changes :).